anki/template/template.py

```diff
@@ -5,6 +5,7 @@
 from anki.template import hint; hint.install()
 
 clozeReg = r"(?s)\{\{c%s::(.*?)(::(.*?))?\}\}"
```

Comment:
This is not part of the changes, but @dae it would help legibility if those groups were named (esp. since they're used quite far from their definition)